code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -104,8 +119,14 @@
 			m_type = 1
 			message = "<B>\The [src]</B> does a flip!"
 			src.SpinAnimation(5,1)
+		if("superflip")
+			m_type = 1
+			message = "<B>The [src]</B> starts flipping!"
+			src.reagents.add_reagent("fliptonium", 100)
```

Comment:
implied `src.`

also bad. adding reagents to them could be used to fill their reagent holder and make them immune to chemical attacks.